openapi_tester/loaders.py

```diff
@@ -167,6 +169,69 @@ def resolve_path(self, endpoint_path: str) -> tuple:
             )
         raise ValueError(f"Could not resolve path `{endpoint_path}`")
 
+    @staticmethod
```

Comment:
this logic should not be moved to the loaders, it should remain part of the tester and that the only place where its used. Please stop refactoring stuff just for the sake of refactoring it.